resources/rest-service/cloudify/types/types.yaml

```diff
@@ -856,8 +856,6 @@ workflows:
   scale:
     mapping: default_workflows.cloudify.plugins.workflows.scale_entity
     is_cascading: false
-    availability_rules:
-      node_instances_active: [ 'all', 'partial' ]
```

Comment:
hmm, I think this was correct after all. Why would we want to scale something that isn't installed at all?

Reply:
because "Don't forbid scaling from zero instances"